infrastructure-playbooks/rolling_update.yml

```diff
@@ -79,6 +79,26 @@
 
     - set_fact: rolling_update=true
 
```

Comment:
@leseb and @guits I just add some code here for migrate old fetch dir to new location if not yet exists, which clone similar logic from https://github.com/kubernetes-incubator/kubespray/pull/2345. Hopefully reduce some impact on existing deployments.